test/performance_test/performance.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"time"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+	"github.com/tsenart/vegeta/lib/plot"
+)
+
+// might add a with metrics bool option
+var (
+	appTestToken, workerTestToken, apiHost, proto, endpoint, reportFilePath string
+	encrypt                                                                 bool
+	freq, duration                                                          int
+)
+
+func init() {
+	flag.StringVar(&appTestToken, "api_test_token", "", "access token used for api performance testing")
+	flag.StringVar(&workerTestToken, "worker_test_token", "", "access token used for worker performance testing")
+	flag.StringVar(&apiHost, "host", "localhost:3000", "host to send requests to")
+	flag.IntVar(&duration, "duration", 60, "seconds: the total time to run the test")
+	flag.IntVar(&freq, "freq", 10, "the number of requests per second")
```

Comment:
These duration/freq params need to be exposed to Jenkins, correct?  This will allow us to tweak our perf/load/stress tests all from a single Jenkins job.

Reply:
yep and they get passed in just like the rest of the flags. If they don't get passed in then it will default to the values above

Reply:
Do we need separate vars for app and worker?  
In other words,  if we set `duration` to 60... that means both app and worker run for 60 seconds.  Should we make it more flexible?